WcaOnRails/app/assets/stylesheets/persons.scss

```diff
@@ -0,0 +1,34 @@
+#persons-query-fields {
+  #search-box {
+    margin: 15px 0;
+  }
+}
+
+.persons-table {
+  .name {
+    width: 40%;
+  }
+
+  .wca-id {
+    width: 10%;
+  }
+
+  .country {
+    width: 30%;
+  }
+
+  .competitions-count,
+  .podiums-count {
+    width: 10%;
+    text-align: center;
+  }
+}
+
+.fixed-table-loading,
+.pagination-detail {
+  display: none !important;
```

Comment:
I think better that ignoring this entire file would be to add  `// scss-lint:disable ImportantRule` to the end of this line. Also, a sentence about why we need `!important` here would be appreciated =)